src/isolate.ts

```diff
@@ -12,6 +12,19 @@ interface Mappable<T, R> {
 
 export function isolateSink(sink: any, scope: string): any {
   return <Mappable<VNode, VNode>>sink.map((vTree: VNode) => {
+    if (vTree.data.isolate) {
+      const existingScope =
+        parseInt(vTree.data.isolate.split(SCOPE_PREFIX + 'cycle')[1]);
```

Comment:
This seems too hacky, but I'll merge it for the time being. Since we anyway have control over everything under `vTree.data.isolate` we could also store the integer there.